client/commands.c

```diff
@@ -48,73 +50,221 @@ void processXferCommand(char *nextParam, FILE *out, int ispwd, uint32_t* pwd);
 void processLastBlocksCommand(char *nextParam, FILE *out);
 void processMinersCommand(char *nextParam, FILE *out);
 
+int xdag_com_account(char *, FILE*);
+int xdag_com_balance(char *, FILE*);
+int xdag_com_block(char *, FILE*);
+int xdag_com_lastblocks(char *, FILE*);
+int xdag_com_keyGen(char *, FILE*);
+int xdag_com_level(char *, FILE*);
+int xdag_com_mining(char *, FILE*);
+int xdag_com_net(char *, FILE*);
+int xdag_com_pool(char *, FILE*);
+int xdag_com_miners(char *, FILE*);
+int xdag_com_xfer(char *, FILE*);
+int xdag_com_stats(char *, FILE*);
+int xdag_com_state(char *, FILE*);
+int xdag_com_help(char *, FILE*);
+int xdag_com_run(char *, FILE*);
+int xdag_com_terminate(char *, FILE*);
+int xdag_com_exit(char *, FILE*);
+
+char* xdag_com_generator(const char*, int);
+char** xdag_com_completion(const char *, int, int);
```

Comment:
Function xdag_com_completion is already declared in header file. Not necessary to declare it again.

Reply:
ok